src/main.js

```diff
@@ -1 +1,54 @@
+ // Smooth scroll to sections

```

Comment:
I suggest rewrirte JS code using Event Delegation pattern and using camelCase notation for variables.
Variables shoudn't have any `_` signs and should be clearly names (not **prj** or **cont**)